server/src/db/migrations/20220126063347_holds_table.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from "knex";
+
+
+export async function up(knex: Knex): Promise<void> {
+    return knex.schema.hasTable('Holds').then(function(exists) {
+        if (!exists) {
+            return knex.schema.createTable('Holds', function(t) {
+                t.increments('id');
+                t.enu('type', ['INCOMPLETE_TRAINING', 'SAFETY_VIOLATION', 'BALANCE_DUE']);
+                t.text('description');
+                t.integer('created_by').references('id').inTable('Users');
```

Comment:
There might be situations where a hold is just automatically given to someone (I could be wrong but I vaguely remember a conversation), in that scenario the `created_by` field is a little awkward.